react/react_test.go

```diff
@@ -192,6 +197,37 @@ func TestCallbackAddRemove(t *testing.T) {
 	}
 }
 
+func TestMultipleCallbackRemoval(t *testing.T) {
+	r := New()
+	inp := r.CreateInput(1)
+	c := r.CreateCompute1(inp, func(v int) int { return v + 1 })
+
+	numCallbacks := 5
```

Comment:
why 5? no real reason; it was arbitrary. 2 is enough to trigger the problem that @soniakeys wanted me to trigger, really.